### ISSUE TITLE
remove desiutil dependency

### ISSUE THREAD

**Issue by sbailey** [desihub/specter#20]
specter should be experiment agnostic, and thus it shouldn't depend upon desiutil (even if desiutil is open source).  Currently it depends upon desiutil in setup.py:
```
from desiutil.setup import DesiTest, DesiVersion, get_version
```
Understand the implications for desiInstall prior to removing this.

**Comment by weaverba137** [comment]
Changing this will not affect desiInstall, although the setup.py file will need some alternate method to set the version string.

I notice that specsim uses the astropy infrastructure as an alternative. @dkirkby, can you tell us if specsim is following the astropy-affiliated package specifications in detail, or did you just pick an choose a few items from the astropy infrastructure?

**Comment by dkirkby** [comment]
I am trying to follow them in detail, although I haven't taken the final step of actually registering the package within the astropy ecosystem.

Regarding the version string, the specsim `setup.py` includes:
```
# VERSION should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
VERSION = '0.2dev'

# Indicates if this version is a release version
RELEASE = 'dev' not in VERSION

if not RELEASE:
    VERSION += get_git_devstr(False)
```
which leads to strings like `0.2dev263`.

**Comment by weaverba137** [comment]
OK, @dkirkby that's what I needed to know.  For the record, that *is* the standard method of setting the version on astropy-affiliated packages, and I know it works.  But you have to set it by hand.  This requires extra-special care when creating a tag.

Since we already have one astropy-affiliated package in our ecosystem, adding another shouldn't be a problem.

**Comment by dkirkby** [comment]
They provide a straightfoward [checklist](http://astropy.readthedocs.org/en/latest/development/affiliated-packages.html#releasing-an-affiliated-package) that helps minimize tagging errors (which aren't too serious anyway since tags are not part of the read-only history).

**Comment by weaverba137** [comment]
Still waiting on an answer on the astropy-affiliated question.

**Comment by dkirkby** [comment]
Can you remind us what the outstanding question is?

**Comment by weaverba137** [comment]
Should specter switch to being an astropy-affiliated package just like specsim?  Because we can't use the desiutil infrastructure for specter.

**Comment by weaverba137** [comment]
specter hasn't been tagged in a while, and it would be nice to resolve this question so that the next tag is easily installable.

**Comment by sbailey** [comment]
desiutil dependency removed as part of PR #30.
